pennylane/devices/tests/test_properties.py

```diff
@@ -71,23 +70,29 @@ def test_load_device(self, device_kwargs):
         device_kwargs["shots"] = 1234
 
         dev = qml.device(**device_kwargs)
+        if isinstance(dev, qml.devices.Device):
+            pytest.skip("test is old interface specific.")
```

Comment:
can you add a similar test for the new device API, or at least an `if-else` block in this test? Only curious because the default `__repr__` method assumes the device defines `shots` and `wires` properties

Reply:
Writing one up.